tools/style.sh

```diff
@@ -7,7 +7,7 @@ then
 fi
 
 Rscript -e "if (!require('styler')) install.packages('styler')"
-#Rscript -e 'styler::style_pkg(exclude_files = list.files("./R", pattern = "^gen-*.*|^RcppExports.*"))'
+Rscript -e 'styler::style_pkg(exclude_files = list.files("./R", pattern = "^gen-*.*|^RcppExports.*"))'
```

Comment:
I think we need `full.names = TRUE` because otherwise it will look for the file name in the root directory.

```suggestion
Rscript -e 'styler::style_pkg(exclude_files = list.files("./R", pattern = "^gen-*.*|^RcppExports.*", full.names = TRUE))'
```